### PR TITLE
Activity Export API doc updates

### PR DESCRIPTION
**Pull request by ccapurso**
### Description
Add documentation updates for the activity export API.

RFC: [VLT-324: Enhance Activity Export API with Identity Metadata](https://docs.google.com/document/d/1L42mm7rSwJSwMN4h7TnJCwt2GJ_DjKnlJIW8LxJzRgU/edit)
Jira: [VAULT-28038](https://hashicorp.atlassian.net/browse/VAULT-28038)

### TODO only if you're a HashiCorp employee
- [x] **Backport Labels:** If this PR is in the ENT repo and needs to be backported, backport  
  to N, N-1, and N-2, using the `backport/ent/x.x.x+ent` labels. If this PR is in the CE repo, you should only backport to N, using the `backport/x.x.x` label, not the enterprise labels.
    - [x] If this fixes a critical security vulnerability or [severity 1](https://www.hashicorp.com/customer-success/enterprise-support) bug, it will also need to be backported to the current [LTS versions](https://developer.hashicorp.com/vault/docs/enterprise/lts#why-is-there-a-risk-to-updating-to-a-non-lts-vault-enterprise-version) of Vault. To ensure this, use **all** available enterprise labels.
- [x] **ENT Breakage:** If this PR either 1) removes a public function OR 2) changes the signature
  of a public function, even if that change is in a CE file, _double check_ that
  applying the patch for this PR to the ENT repo and running tests doesn't
  break any tests. Sometimes ENT only tests rely on public functions in CE
  files.
- [x] **Jira:** If this change has an associated Jira, it's referenced either
  in the PR description, commit message, or branch name.
- [x] **RFC:** If this change has an associated RFC, please link it in the description.
- [x] **ENT PR:** If this change has an associated ENT PR, please link it in the
  description. Also, make sure the changelog is in this PR, _not_ in your ENT PR.

[VAULT-28038]: https://hashicorp.atlassian.net/browse/VAULT-28038?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by vishalnayak — APPROVED** [review]

File: website/content/api-docs/system/internal-counters.mdx [hunk 1]
```diff
@@ -1088,8 +1088,12 @@ months in the requested time range.
 information returned by this API will include activity for this month, however
 it may be up to 20 minutes delayed.
 
+- This endpoint can be called from all namespaces. The requested namespace (specified via the `X-Vault-Namespace`
+  header or as part of the path) will act as a filter. The exported data will include activity for the requested
```
Comment (VioletHynes) [inline website/content/api-docs/system/internal-counters.mdx:1092]:
```suggestion
- This endpoint can be called from all namespaces. The requested namespace will act as a filter. 
The exported data will include activity for the requested
```

File: website/content/api-docs/system/internal-counters.mdx [hunk 2]
```diff
@@ -1088,8 +1088,12 @@ months in the requested time range.
...
+  namespace and all of its children.
```
Comment (VioletHynes) [inline website/content/api-docs/system/internal-counters.mdx:1093]:
```suggestion
- This endpoint can be called from all namespaces. The requested namespace will act as a filter. The exported data will include activity for the requested
  namespace and all of its children.
```

Comment (VioletHynes) [reply website/content/api-docs/system/internal-counters.mdx:1093]: There are other ways for namespaces to be selected, such as the chroot namespace listener functionality -- I'd rather not enumerate them all here and we may add more in the future

**Review by VioletHynes — APPROVED** [review]
Looks good! Just a little comment that's a take-it-or-leave-it one.

File: website/content/api-docs/system/internal-counters.mdx [hunk 3]
```diff
@@ -1088,8 +1088,12 @@ months in the requested time range.
 information returned by this API will include activity for this month, however
```
Comment (divyaac) [inline website/content/api-docs/system/internal-counters.mdx:1088]: should we add something about that -raw flag that is necessary to output the data in the CLI?

File: website/content/api-docs/system/internal-counters.mdx (hunk 2 above)
Comment (ccapurso) [reply website/content/api-docs/system/internal-counters.mdx:1093]: That's a great point, thanks for the feedback!

File: website/content/api-docs/system/internal-counters.mdx (hunk 3 above)
Comment (ccapurso) [reply website/content/api-docs/system/internal-counters.mdx:1088]: Ah, yes. That's a great point. I was thinking of adding that separate from the API docs. Possibly here https://developer.hashicorp.com/vault/docs/concepts/client-count.

**Pull request merged**